assertions/screenshotIdenticalToBaseline.js

```diff
@@ -53,7 +57,12 @@ exports.assertion = function screenshotIdenticalToBaseline(
                 compareWithBaseline(this.api, screenshot, fileName, settings).then((result) => {
                     comparisonResult = result ? result : result.value
                     if(result.value === true && result.diff !== result.threshold) {
-                        this.message = `The difference between the screenshots for <${elementId.selector}> was ${result.diff} when ${result.threshold} was expected`
+                        if (elementId.selector) {
+                            elementId = elementId.selector
+                        } else {
+                            elementId = elementId;
+                        }
+                        this.message = `The difference between the screenshots for <${elementId}> was ${result.diff} when ${result.threshold} was expected`
```

Comment:
thanks for adding this message, I did the same in local

Reply:
Is undefined not falsey?

Reply:
Based on https://github.com/bbc/nightwatch-vrt/blob/d66fd5bce4a9397e26e38d24b9697fcdf34461bc/lib/compare-with-baseline.js#L117 I think the `result.diff` is already the `diff.percent` value fixed to 2dp